Utilities/Scripts/SlicerWizard/__version__.py

```diff
@@ -1,6 +1,6 @@
 __version_info__ = (
-  4,
-  9,
+  1,
+  1,
```

Comment:
Was this change intentional? It does not seem to be necessary.

Reply:
It was probably not. But JC was debugging stuff on my computer while I was having lunch :smile: so I thought I would commit all the changes. @jcfr said he will do additional testing and clean up the fix.